app/src/main/java/com/github/alfabravo2013/readyforexams/domain/home/LoadChecklistUseCase.kt

```diff
@@ -0,0 +1,11 @@
+package com.github.alfabravo2013.readyforexams.domain.home
+
+import com.github.alfabravo2013.readyforexams.domain.models.toChecklistRepresentation
+import com.github.alfabravo2013.readyforexams.presentation.models.ChecklistRepresentation
+
+class LoadChecklistUseCase(private val checklistRepository: ChecklistRepository) {
+    fun getChecklists(): List<ChecklistRepresentation> =
+        checklistRepository.getChecklists().map { checklist ->
```

Comment:
You can leave this here for now and change it in the future but mapping should be done by the repository usually.